sys-apps/ndctl/ndctl-65.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools bash-completion-r1 linux-info systemd
+
+DESCRIPTION="Utility library for managing the libnvdimm sub-system in the Linux kernel"
+
+HOMEPAGE="http://pmem.io/"
+SRC_URI="https://github.com/pmem/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+# All the tests, like the rest of the nvdimm subsystem, require root to run
```

Comment:
src_test can be run as root, so this doesn't make sense to be given as a reason to restrict tests.

Reply:
The test still need special development versions of the nvdimm module, and furthermore the sandbox is on in src_test, so I can't see any way the test would ever work in Portage

But there may be a compromise: There's an --enable-test configure flag (currently not passsed) that will embed the test into ndctl, they can then be run any time using `ndctl test` . If this would be desirable i can enable it, but should it be gated behind a USE flag and if so what should it need named?